backends/dpdk/printUtils.cpp

```diff
@@ -0,0 +1,121 @@
+/* Copyright 2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "printUtils.h"
+
+namespace DPDK {
+
+bool ConvertToString::preorder(const IR::Expression *e) {
+    BUG("%1% not implemented", e);
+    return false;
+}
+
+bool ConvertToString::preorder(const IR::Type *t) {
+    std::cerr << t->node_type_name() << std::endl;
```

Comment:
You know that BUG can display a message?
You can add a format string `BUG("not implemented %1%", p->node_type_name());`
Don't use std::cerr.

Reply:
Updated as per suggestion.